extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -514,6 +532,29 @@ private static OidcConfigurationMetadata createLocalMetadata(OidcTenantConfig oi
                 oidcConfig.token.issuer.orElse(null));
     }
 
+    private static void fireOidcServerNotAvailableEvent(String authServerUrl, String tenantId) {
+        if (fireOidcServerEvent(authServerUrl, OIDC_SERVER_NOT_AVAILABLE)) {
+            tenantsForWhichWeReportServerAvailable.add(tenantId);
```

Comment:
I see. I wonder if the time come to introduce OIDC SecurityEvent subtypes ? May be later though

Reply:
I'm honestly not sure where to draw a line. It all depends on how / how-much will users use this. I don't want to add too much of a code to Quarkus OIDC. Considering this event is not fired for every application, but only those in (temporary?) problems, I think cost of this PR is going to be very small. So I'd wait.

Reply:
@michalvavrik OK, lets not do it for now, thanks